backend/__tests__/vendorCreateProperty.test.js

```
@@ -0,0 +1,138 @@
+require('dotenv').config();
+const request = require('supertest');
+const app = require('../server'); // Assuming your Express app is exported from app.js
+const { User } = require('../models/userSchemas');
+const { Property } = require('../models/propertySchemas');
+const path = require('path');
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)